redis/asyncio/connection.py

```diff
@@ -816,7 +820,7 @@ async def on_connect(self) -> None:
             if str_if_bytes(await self.read_response()) != "OK":
                 raise ConnectionError("Invalid Database")
 
-    async def disconnect(self) -> None:
+    async def disconnect(self, *args) -> None:
```

Comment:
@szumka Why did you add the args here? they have no use in the function

Reply:
Thanks for the review. I will remove those args.